docs/software/lumi-o/common-use-cases.md

```diff
@@ -0,0 +1,88 @@
+# Common use cases
+
+## Processing data in LUMI
+
+LUMI supercomputer provides disk environments for working with large datasets e.g. _flash_, _scratch_. These storage areas are however not intended for storing data that is not actively used. For example in the _scratch_ area of LUMI the un-used files are automatically removed after 180 days. 
+
+One of the main use cases of LUMI-O is to store data while it is not actively used in the LUMI supercomputer. When you start working, you stage in the data from LUMI-O. And when the data is no longer actively used, it can be staged out to LUMI-O. 
+
+In LUMI, connection to LUMI-O can be established with the guide from [Accessing LUMI-O](./accessing-lumi-o.md):
+```text
+source /project/project_462000008/lumi-o/lumi-o_setup.sh
+```
+Make sure you point the source command at the proper destination.
+
+After that you can:
+
+**List the data buckets and objects in LUMI-O:** For listing we recommend a-list:
+```text
+a-list
+```
+The command above lists available data buckets in LUMI-O. To list data objects in a bucket give command:
+```text
+a-list bucket_name
+```
+
+**Copy data from LUMI-O to a supercomputer (stage in):** For downloading we recommend a-get:
+```text
+a-get bucket/object_name
+```
+
+**Copy data from a Supercomputer to LUMI-O (stage out):** For uploading we recommend a-put:
+```text
+a-put filename
+```
+
+## Sharing data
+
+Sharing data, e.g. datasets or research results, is easy in the object storage. You can share these either with a limited audience, e.g. other projects, or allow access for everybody by making the data public.
+
+The data can be accessed and shared in a variety of ways:
+
+* **Private – default:** By default, if you do not specify anything else, the contents of buckets can only be accessed by authenticated members of your project. 
+
+* **Access control lists:** Access control lists (ACLs) work on buckets, not objects. With ACLs, you can share your data in a limited manner to other projects. You can e.g. grant a collaboration project authenticated read access to your datasets.
+ 
+ * **Public:** You can also have ACLs granting public read access to data, which is useful e.g. for permanently sharing public scientific results or public datasets.
+
+## Static web content
+
+A common way to use the object storage is storing static web content, such as images, videos, audio, pdfs or other downloadable content, and adding links to it on a web page, which can run either inside LUMI or somewhere else, [like this example](https://a3s.fi/my_fishbucket/my_fish).
```

Comment:
Maybe example link would be better suitable with LUMI-O based bucket
https://462000002.lumidata.eu/get44fa4ddd34345/